### PR TITLE
Iterate over static files, too?

### PR DESCRIPTION
**Pull request by gjtorikian**
Maybe fixes https://github.com/gjtorikian/jekyll-last-modified-at/issues/33. 

@pathawks Would it be possible for you to use this branch to test whatever it is you are testing?

### CODE REVIEW

**Comment by pathawks** [comment]
```
     Failure/Error: site.process
     NoMethodError:
       undefined method `data' for #<Jekyll::StaticFile:0x007f8cbc0cd410>
```

**Comment by parkr** [comment]
It would be nice to have this data available for static files.

File: lib/jekyll-last-modified-at/generator.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module LastModifiedAt
     class Generator < Jekyll::Generator
 
       def generate(site)
-        %w(posts pages docs_to_write).each do |type|
+        %w(posts pages static_files docs_to_write).each do |type|
           site.send(type).each do |item|
             item.data['last_modified_at'] = Determinator.new(site.source, item.path)
```
Comment (parkr) [inline lib/jekyll-last-modified-at/generator.rb]: In order to support future versions of Jekyll, keep this change and change this line to:

```ruby
item.data['last_modified_at'] = Determinator.new(site.source, item.path) if item.respond_to?(:data)
```

**Comment by gjtorikian** [comment]
Stale, as original issue was closed.

**Pull request closed without merging**